src/app/campaign/store/reducers/flight.reducer.ts

```diff
@@ -125,7 +125,7 @@ const _reducer = createReducer(
       newState = adapter.addMany(
         newFlightIds.map(newId => {
           const flight = docToFlight(createdFlightDocs[newId]);
-          return { localFlight: flight, remoteFlight: flight, changed: false, valid: true };
+          return { doc: createdFlightDocs[newId], localFlight: flight, remoteFlight: flight, changed: false, valid: true };
```

Comment:
The campaign.service gets the haldoc from the store in order to call update against it. https://github.com/PRX/grove.prx.org/blob/master/src/app/core/campaign/campaign.service.ts#L82
But the CampaignSaveSuccess was not saving the haldoc with created flights, whoops!

Fixed _forever_